### PR TITLE
Add videos and slides

### PR DESCRIPTION
**Pull request by nerucheva**
**Describe what changes this pull request brings**
This PR removes the schedule and adds a section with links to videos and slides for talks.

**Steps to test**
1. Open [the page](https://deploy-preview-53--ebpf-summit-2021.netlify.app/summit-2022/)
2. Confirm that everything looks and works as expected

<details> 
<summary>Screenshots</summary>

image 
</details> 

**References**

1. [Design](https://www.figma.com/file/L49LK6NTH0A5PVwGSW25ub/EBPF-Summit?node-id=2252%3A4664)

### CODE REVIEW

File: src/components/pages/summit-2022/talks/talks.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                  Watch
+                </Button>
+              </div>
+            </li>
+          ))}
+        </ul>
+        {!isFullListShown && (
+          <div className="mt-14  text-center">
+            <Button
+              className="text-xl text-[#FF9800] transition-colors duration-200 hover:text-[#FFAD33]"
```
Comment (vannyle) [inline src/components/pages/summit-2022/talks/talks.jsx:57]: This color `[#FF9800]` should be added to tailwind config

File: src/components/pages/summit-2022/talks/talks.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                  to={youtubeLink}
+                >
+                  Watch
+                </Button>
+              </div>
+            </li>
+          ))}
+        </ul>
+        {!isFullListShown && (
+          <div className="mt-14  text-center">
```
Comment (vannyle) [inline src/components/pages/summit-2022/talks/talks.jsx:55]:
```suggestion
          <div className="mt-14 text-center">
```

File: src/components/pages/summit-2022/talks/talks.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+              key={index}
+            >
+              <div className="space-y-2.5">
+                <span className="leading-none text-gray-1">{speaker}</span>
+                <h3 className="text-xl font-bold leading-snug md:text-lg max-w-[696px]">{topic}</h3>
+              </div>
+              <div className="flex ml-12 lg:ml-0 items-center space-x-4 lg:space-x-0 lg:flex-row-reverse lg:items-center lg:space-y-0">
+                {pdfLink && (
+                  <Button
+                    className="items-center h-11 leading-none lg:h-10 md:text-base lg:ml-5 rounded-lg py-3.5 px-5 text-base"
```
Comment (vannyle) [inline src/components/pages/summit-2022/talks/talks.jsx:30]:
```suggestion
                    className="items-center h-11 leading-none lg:h-10 md:text-base lg:ml-5 rounded-lg py-3.5 px-5"
```

File: src/components/pages/summit-2022/talks/talks.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                    theme="gray"
+                    size="xs"
```
Comment (vannyle) [inline src/components/pages/summit-2022/talks/talks.jsx:32]:
```suggestion
                    size="sm"
```

File: src/pages/summit-2022.jsx [hunk 5]
```diff
@@ -78,6 +80,11 @@ const lastYear = {
   ],
 };
 
+const talks = {
+  title: 'Talks',
+  items: talksData.filter((item) => !item.keynotes),
```
Comment (vannyle) [inline src/pages/summit-2022.jsx:85]: What is this filter for? I couldn't find the item with keynotes at all

File: src/components/pages/summit-2022/talks/talks.jsx [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    to={pdfLink}
+                  >
+                    Download PDF
+                  </Button>
+                )}
+                <Button
+                  className="rounded-lg py-3.5 leading-none px-5 text-base"
```
Comment (vannyle) [inline src/components/pages/summit-2022/talks/talks.jsx:41]: No need to overwrite the font-size once you have size for the button

**Review by vannyle — CHANGES_REQUESTED** [review]
The PR looks great, just a few nitpick comments

**Review by vannyle — APPROVED** [review]
Nice job! 💯

**Review by k8isdead — APPROVED** [review]

**Pull request merged**